### ISSUE TITLE
Host binary using GitHub Releases instead of Dropbox

### ISSUE THREAD

**Issue by tlhunter** [MrCheeze/GYTB#30]
By hosting the binary via GitHub release it's one less site to visit and one less user account to create.

**Comment by egscodes** [comment]
I agree, in case the Dropbox link dies the app is available in universal updater (sadly both links cant be saved in the web archive)
